docs/containers-registries.conf.5.md

```diff
@@ -99,7 +99,7 @@ as-is. But other settings like insecure/blocked/mirrors will be applied to match
 
 `mirror`
 : An array of TOML tables specifying (possibly-partial) mirrors for the
-`prefix`-rooted namespace.
+`prefix`-rooted namespace (i.e., the previous registry).
```

Comment:
Is that the “current” registry, actually?

It’s not even a _“registry”_ at all, it’s that `prefix`-delineated space, which might be smaller than a registry.

How about “the current `[[registry]]` TOML table`?